lib/generators/hyrax/batch_ingest/install_generator.rb

```diff
@@ -10,6 +10,10 @@ class InstallGenerator < Rails::Generators::Base
       def add_routes
         route "mount Hyrax::BatchIngest::Engine, at: '/'"
       end
+
+      def install_migrations
+        rake 'hyrax:batch_ingest:install:migrations'
+      end
```

Comment:
@cjcolvar we moved this to here from the test app generator, because it's part of the general installation in all contexts, not just tests.